include/spdlog/details/logger_impl.h

```diff
@@ -87,6 +88,9 @@ inline void spdlog::logger::log(level::level_enum lvl, const char* msg)
     {
         details::log_msg log_msg(&_name, lvl);
         log_msg.raw << msg;
+        if (_formatter->do_msg_count()) {
```

Comment:
I think we can do both:

1. Just use atomic increment without the conditional logic
2. Disable by default, and have the option to enable it in tweakme.h with
```c++
#define SPDLOG_ENABLE_MESSAGE_COUNTER
```
If in the future there will be more demand for this, I will enable by default
3. Need to document in the wiki near the %i  option that it is enabled only if defined in tweakme